code/modules/mob/inventory.dm

```diff
@@ -485,6 +485,19 @@
 
 	DEFAULT_QUEUE_OR_CALL_VERB(VERB_CALLBACK(src, PROC_REF(execute_quick_equip)))
 
+/// Safely drop everything, without deconstructing the mob
+/mob/proc/drop_everything(del_on_drop, force, del_if_nodrop)
```

Comment:
One bitflag is better than multiple boolean arguments. We should be transiting to this style long-term.

See:
- https://github.com/tgstation/tgstation/pull/78754
- https://github.com/tgstation/tgstation/pull/64124
- https://github.com/tgstation/tgstation/pull/73434

Reply:
perhaps another time